### PR TITLE
Chore/account factory refactoring

### PR DESCRIPTION
**Pull request by Aboudjem**
- Fix Linter errors
- Fix typos
- Add Natspec
- Rename tests
- Removed unused imports

### CODE REVIEW

File: contracts/factory/K1ValidatorFactory.sol [hunk 1]
```diff
@@ -66,10 +66,10 @@ contract K1ValidatorFactory is BootstrapUtil, Stakeable {
 
         (bool alreadyDeployed, address account) = LibClone.createDeterministicERC1967(msg.value, ACCOUNT_IMPLEMENTATION, actualSalt);
         BootstrapConfig memory validator = _makeBootstrapConfig(K1_VALIDATOR, abi.encodePacked(eoaOwner));
-        bytes memory _initData = BOOTSTRAPPER._getInitNexusWithSingleValidatorCalldata(validator);
+        bytes memory initData = BOOTSTRAPPER.getInitNexusWithSingleValidatorCalldata(validator);
```
Comment (livingrockrises) [inline contracts/factory/K1ValidatorFactory.sol:69]: was thinking to rename this to accountInitData or accountBootstrapData

File: contracts/interfaces/common/IStakeable.sol [hunk 2]
```diff
@@ -12,20 +12,24 @@ pragma solidity ^0.8.24;
...
-    ) external payable;
+    /// @notice Stakes a certain amount of Ether on an EntryPoint.
+    /// @dev The contract should have enough Ether to cover the stake.
+    /// @param epAddress Address of the EntryPoint where the stake is added.
+    /// @param unstakeDelaySec The delay in seconds before the stake can be unlocked.
+    function addStake(address epAddress, uint32 unstakeDelaySec) external payable;
 
+    /// @notice Unlocks the stake on an EntryPoint.
+    /// @dev This starts the unstaking delay after which funds can be withdrawn.
+    /// @param epAddress Address of the EntryPoint where the stake is unlocked.
```
Comment (livingrockrises) [inline contracts/interfaces/common/IStakeable.sol:27]: same here
you can copy this from StakeManager.sol of infinitism repo

File: contracts/factory/K1ValidatorFactory.sol (hunk 1 above)
Comment (Aboudjem) [reply contracts/factory/K1ValidatorFactory.sol:69]: please check the comment on your branch first and lmk if that makes sense.

File: contracts/interfaces/common/IStakeable.sol (hunk 2 above)
Comment (Aboudjem) [reply contracts/interfaces/common/IStakeable.sol]: done

File: contracts/common/Stakeable.sol [hunk 3]
```diff
@@ -12,53 +12,40 @@ pragma solidity ^0.8.24;
...
 
-    /**
-     * @dev Unlocks the stake on an EntryPoint.
-     * @notice This starts the unstaking delay after which funds can be withdrawn.
-     * @param epAddress Address of the EntryPoint where the stake is unlocked.
-     */
-    function unlockStake(address epAddress) external override onlyOwner {
+    /// @notice Unlocks the stake on an EntryPoint.
+    /// @dev This starts the unstaking delay after which funds can be withdrawn.
+    /// @param epAddress Address of the EntryPoint where the stake is unlocked.
```
[the remark this answers is not in the document]
Comment (Aboudjem) [reply contracts/common/Stakeable.sol]: done !

**Review by livingrockrises — APPROVED** [review]
lgtm 
left review

**Comment by Aboudjem** [comment]
> lgtm left review

fixed

**Pull request merged**